experimental/rsocket/RSocketErrors.h

```diff
@@ -0,0 +1,133 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
+
+#include <string>
+
+namespace rsocket {
+
+/*
+ * Error Codes from
+ * https://github.com/ReactiveSocket/reactivesocket/blob/master/Protocol.md#error-codes
+ */
+class RSocketError : public std::runtime_error {
+ public:
+  explicit RSocketError(const std::string& s)
+      : std::runtime_error(std::move(s)) {}
```

Comment:
Right, and I remember reading that having const there also prevents moving so will remove that.